packages/cadl-ranch-specs/cadl-ranch-summary.md

```diff
@@ -791,7 +791,7 @@ Expected response body:
 }
 ```
 
-### Azure_ResourceManager_Models_CommonTypes_ManagedIdentity_ManagedIdentityTrackedResources_createWithSystemAssigned
+### Azure_ResourceManager_Models_CommonTypes_ManagedIdentity_CrudOperations_createWithSystemAssigned
```

Comment:
Original intent for the extra interface `CrudOperations` was to mitigate breaking changes to client code, as we previously have `ManagedIdentityTrackedResources`, which is definitely inappropriate...

Though a further look, as weidong mentioned, `create` directly after it may lead to confusion, even if we remove the interface, e.g. 

`Azure_ResourceManager_Models_CommonTypes_ManagedIdentity_createWithSystemAssigned`

Looks like it's for CRUD operations on ManagedIdentity. Though actually, it's rather for creating another resource with ManagedIdentity as its property.

Thus, we proposed:
`Azure_ResourceManager_Models_CommonTypes_ManagedIdentity_AsResourceProperty_createWithSystemAssigned`
or 
`Azure_ResourceManager_Models_CommonTypes_ManagedIdentity_ResourceProperty_createWithSystemAssigned`.

Or we could remove the managed identity test case, and embed it into one of the resources test... Downside is that we may mix too many test cases inresources...